docs/advanced-features/using-mdx.md

```diff
@@ -158,9 +174,10 @@ export default function Post(props) {
 }
 ```
 
-## Helpful links
+## Helpful Links
 
 - [MDX](https://mdxjs.com)
 - [`@next/mdx`](https://www.npmjs.com/package/@next/mdx)
 - [remark](https://github.com/remarkjs/remark)
 - [rehype](https://github.com/rehypejs/rehype)
+- [gray-matter: Adding frontmatter to your MDX file](https://github.com/jonschlinkert/gray-matter)
```

Comment:
I'm not sure we add this here because to use frontmatter in MDX you have to use the JS object as shown above. To use gray matter, you need a remark or rehype plugin. I would say we could link there, but that might be more confusing.

Reply:
That's fair. I'll move the link to the relevant section